conda.md

```diff
@@ -35,37 +35,41 @@ Prompt/WSL prompt on Windows). If you are on Windows, I highly recommend using W
 
 3.  So first of all, let's try to create a new, "clean" environment.
     Type `conda create -n bestpractices`. This will create a new
-    environment named "bestpractices". Go ahead and do `conda activate
-    bestpractices` - this should change the beginning of your prompt
-    to `(bestpractices)`. We're now inside this new environment.
+    environment named "bestpractices" (`-n` is shorthand for `--name`).
+    Go ahead and do `conda activate bestpractices` - this should change
+    the beginning of your prompt to `(bestpractices)`. We're now inside
+    this new environment.
 
-4.  Try running `python`. It won't work. Why?
+5.  Try running `python`. It probably didn't work. Why?
 
-5.  Let's deactivate this empty environment and delete it. Do `conda
-    deactivate` and `conda remove --all -n bestpractices`. Try running
+6.  Let's deactivate this empty environment and delete it. Do `conda
+    deactivate` and `conda remove -n bestpractices --all`. Try running
```

Comment:
This order is based on the conda docs.
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#removing-an-environment
Also I think it's less scary looking, because `conda remove --all` could make someone think all envs will be removed or something.